sssom/parsers.py

```diff
@@ -247,6 +234,7 @@ def from_sssom_rdf(
     curie_map = _check_curie_map(curie_map)
 
     if mapping_predicates is None:
+        # FIXME unused
```

Comment:
@matentzn , thoughts? Would this be unused all the time?

Reply:
This doesn't necessarily have to be addressed in this PR - I only added the FIXME tag for us to check in on later